tools/BuildAssets/targets/signing.targets

```diff
@@ -65,12 +65,16 @@
       <SignOperationLogDir>$(BinariesFolder)</SignOperationLogDir>
     </PropertyGroup>
   </Target>
-  <Target Name="SignNugetPackage" DependsOnTargets="SignNugetPackageDependsOn" Condition=" '$(CodeSign)' == 'true' AND '$(Configuration)' == 'Release' ">
+  <Target Name="SignNugetPackage" DependsOnTargets="SignNugetPackageDependsOn" 
+          Condition=" '$(CodeSign)' == 'true' AND '$(Configuration)' == 'Release' AND '$(SkipAuthSign)' != 'true' ">
```

Comment:
Any reason to have multiple properties to control signing? I.e. CodeSign and SkipAuthSign?

Reply:
Signing has come along a lot of iteration. First it was Authenticode signing using old singing queues. Then came the new queue but only for Nuget packages.
Now everything is on the new queue.
Not to mention, there are other repos who have some legacy way to sign components.
So as of now everything is controlled using these two variables.
I will have to do a thorough review if anyone is using CodeSign (legacy property that I inherited)
But yes I agree, these are redundant properties and this should be cleaned up.